### PR TITLE
Feature/glossary redesign

### PR DESCRIPTION
**Pull request by mjoyce91**
Relies on #237

Redesigns the Glossary. Completes TM-1016.

### CODE REVIEW

File: src/Components/Glossary/GlossarySearch/GlossarySearch.jsx [hunk 1]
```diff
@@ -17,8 +17,11 @@ const GlossarySearch = ({ changeText, searchTextValue }) => (
       value={searchTextValue}
       labelSrOnly={false}
       placeholder="Search for terms"
+      inputProps={{
+        autoComplete: 'off',
+      }}
     />
-    <div className="glossary-search-sub">Examples: Tandem, Tour of duty</div>
+    <div className="glossary-search-sub">Example: Tandem</div>
```
Comment (burgwyn) [inline src/Components/Glossary/GlossarySearch/GlossarySearch.jsx]: Let's put Tandem in the base glossary

File: src/sass/_glossary.scss [hunk 2]
```diff
@@ -75,6 +101,7 @@
     position: absolute;
     right: 0;
     top: 0;
+    z-index: 1;
```
Comment (burgwyn) [inline src/sass/_glossary.scss]: Let's make the `z-index` a variable

File: src/Components/Glossary/GlossarySearch/GlossarySearch.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/Glossary/GlossarySearch/GlossarySearch.jsx]: Like in the database?

Comment (burgwyn) [reply src/Components/Glossary/GlossarySearch/GlossarySearch.jsx]: I feel like we could be better to ensure that when we provide a callout to a sample entry that we make sure that entry is available.

**Pull request merged**